script.js

```diff
@@ -55,7 +59,7 @@ addButton.addEventListener('click', () => {
   const bookAuthor = document.getElementById('author').value;
   const book = new Book(bookTitle, bookAuthor);
   shelf.addBook(book);
-  listedBook.insertAdjacentHTML('beforeend', `<li>${book.title} by ${book.author} <button class="remove-button">Remove</button></li>`);
+  listedBook.insertAdjacentHTML('beforeend', `<li class=new-book>${book.title} by ${book.author} <button class="remove-button">Remove</button></li>`);
 });
 
 booksSection.addEventListener('click', (e) => {
```

Comment:
- [ ] When I click on "Remove", the book is not removed and I get this error on the browser console.
<img width="572" alt="Screen Shot 2021-11-25 at 6 00 28 PM" src="https://user-images.githubusercontent.com/47994600/143485860-0f97bd02-ad97-4bda-ba65-ee1890922a17.png">